.github/workflows/staging_build_and_deploy.yml

```diff
@@ -28,6 +28,12 @@ jobs:
         id: variables_population
         run: |
           echo "DOMAIN=${{env.DOMAIN}}
+          GF_SECURITY_ADMIN_PASSWORD=${{secrets.GF_SECURITY_ADMIN_PASSWORD}}
+          GF_SECURITY_ADMIN_USER=${{secrets.GF_SECURITY_ADMIN_USER}} 
+          ALLOY_CONFIG_FILE=config.alloy
+          ALLOY_CONFIG_PATH=/etc/alloy
+          ALLOY_CONFIG_PATH_LOCAL=./infra/alloy/config
+          ALLOY_API_KEY=${{secrets.ALLOY_API_KEY}}
```

Comment:
**suggestion (non-blocking):** These workflows could be more DRY with a single file for common steps

Reply:
Not what I meant, but non-blocking anyway